core/src/toga/style/pack.py

```diff
@@ -118,15 +118,22 @@ def apply(self, prop: str, value: object) -> None:
                 "font_variant",
                 "font_weight",
             ):
-                self._applicator.set_font(
-                    Font(
+                # For other properties, a backend that doesn't support it will simply do
+                # a no-op when instructed to set it. But for font, we need to know
+                # up-front, because just creating a Font object will fail.
```

Comment:
It was for Textual, and it never came up before because Textual's Widget base initializer didn't call a style reapply. With this PR, the core initializer reapplies the style, meaning it tries it for all tested backends — including Textual, which threw a NotImplemented error. (I'm sure it would for web too, if it were tested at all.)

I don't know that I have any particular opinion on whether it's better to check in the applicator or require Font in the backend. The latter seems somewhat arbitrary, but it would streamline this bit in core.

Reply:
I'm not sure I'd say requiring a representation of Font is arbitrary; Textual is the only platform I can think of that doesn't have a strong concept of a font - and even there, there are console variants like oblique and bold. We require all platforms to have a concept of Window; I don't think requiring a stub concept of Font is that far off the same requirement.

Reply:
Fair enough, that makes sense. Would a new requirement like that go under a removal changenote?

Reply:
No - there aren't any alternate backends, so it's only a concern for us internally at this point.